### PR TITLE
Linux only CVE re-build 2.32.2.20240516

### PR DESCRIPTION
**Pull request by swapneils**
CVE Rebuild for Amazon Linux

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 2.32.2.20240516 Linux re-build
+
+*This release has the same Fluent Bit contents as 2.32.0, and is simply a linux-only re-build for recent patches in dependencies installed in the image. There are no windows images for this release.*
```
Comment (matthewfala) [inline CHANGELOG.md]: could you write "...contents as 2.32.2". That's really the same but it's more appropriate to use the most recent version.

**Review by bendu — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 2.32.2.20240516 Linux re-build
+
+*This release has the same Fluent Bit contents as 2.32.0, and is simply a linux-only re-build for recent patches in dependencies installed in the image. There are no windows images for this release.* 
+* Amazon Linux Base: [2.0.20240503.0](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-20240508.html)
+
+
```
Comment (matthewfala) [inline CHANGELOG.md]: Would you remove the extra indentation, for consistency with the rest of the change log?

File: linux.version [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
     "latest": "true",
     "build": "1",
     "fluent-bit": "1.9.10",
     "kinesis-plugin": "v1.10.1",
     "firehose-plugin": "v1.7.1",
     "cloudwatch-plugin": "v1.9.4"
   }
-}
\ No newline at end of file
+}
```
Comment (matthewfala) [inline linux.version]: Would you remove the extra indentation here? It's not the best format, but let's not make unnecessary changes in the versioning file.

**Review by matthewfala — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (swapneils) [reply CHANGELOG.md]: Modified this and the previous CVE release to use 2.32.2.

**Review by bendu — APPROVED** [review]

**Review by PettitWesley — APPROVED** [review]

**Review by matthewfala — CHANGES_REQUESTED** [review]
Would you please squash your commits? Our commit history shouldn't show these minor revisions unless necessary.

**Review by matthewfala — APPROVED** [review]
Looks great, thank you!

**Comment by bendu** [comment]
> Would you please squash your commits? Our commit history shouldn't show these minor revisions unless necessary.

I've gone ahead and changed the PR merge option to "squash" so we no longer have to worry about this
